src/main/java/picard/util/IntervalListTools.java

```diff
@@ -275,24 +275,29 @@
     public boolean INVERT = false;
 
     @Argument(doc = "What value to output to COUNT_OUTPUT file (for scripting)")
-    public Output OUTPUT_VALUE = Output.BASES;
+    public Output OUTPUT_VALUE = Output.NONE;
 
-    @Argument(doc = "File to which to print count of bases or intervals in final output interval list.", optional = true)
+    @Argument(doc = "File to which to print count of bases or intervals in final output interval list.  When this parameter is set OUTPUT_VALUE must be set to a value other than NONE.", optional = true)
     public File COUNT_OUTPUT;
 
     enum Output {
+        NONE {
+            void output(final long totalBaseCount, final long intervalCount, final PrintWriter writer){
+
```

Comment:
throw an error here?

Reply:
With the default changed to print to stdout, it becomes only reasonable to throw an exception here if COUNT_OUTPUT is set.  As written, `customCommandLineValidation` should guarantee that state never reaches here, but I suppose checking and throwing an exception here provides additional safety.